### ISSUE TITLE
Uyuni login screen does not show Uyuni logo

### ISSUE THREAD

**Issue by Bischoff** [uyuni-project/uyuni#3220]
## Version of Uyuni Server and Proxy (if used)
Affects all Uyuni versions.

## Details about issue
It is customary for software products to show their logo on their landing page.

When connecting to Uyuni web interface, the Uyuni logo is not shown.

**Comment by paususe** [comment]
@Bischoff What do you mean? There's not a graphical logo but there is a "text" logo:

![image](https://user-images.githubusercontent.com/50680111/107889545-a3512d00-6f13-11eb-9af9-ee46ba0622e5.png)

**Comment by Bischoff** [comment]
> @Bischoff What do you mean? There's not a graphical logo but there is a "text" logo: (...)

Yes, I meant the graphical logo. It is usual for software to display their graphical "signature".

https://www.uyuni-project.org/img/uyuni-logo.svg

**Comment by paususe** [comment]
Ahhh now I understand

**Comment by jcayouette** [comment]
Yes, we are already using this logo for docs :) @Bischoff Thanks.

**Comment by nodeg** [comment]
Fixed with https://github.com/uyuni-project/uyuni/pull/11430.
